CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-## 2.2.0 2024-07-17
+## 2.3.0 2024-10-14
+### Features
+* Customers can now set their own user agents on the internal clients for audit purposes.
+* Added default user agent for internal clients.
```

Comment:
What do you mean by internal clients here?

Reply:
Clients that we create in the plugin like Sts client, S3Control client, S3Client (for headBucket call)